### PR TITLE
Review nits

### PR DESCRIPTION
**Pull request by MikeBishop**
These changes didn't seem to warrant their own issues or separate PRs.  If there's anything here you'd like to discuss separately, I'm happy to pull it into its own branch.

### CODE REVIEW

File: draft-ietf-httpbis-digest-headers.md [hunk 1]
```diff
@@ -767,12 +766,8 @@ Specification document(s):  {{content-digest}} of this document
 
 The following examples show how representation metadata, payload transformations
 and method impacts on the message and content. When the content
-contains non-printable characters (eg. when it is compressed) it is shown as
-base64-encoded string.
-
-A request with a JSON object without any content coding.
-
-Request:
```
Comment (ioggstream) [inline draft-ietf-httpbis-digest-headers.md:775]: @MikeBishop why remove `Request:`

Comment (MikeBishop) [reply draft-ietf-httpbis-digest-headers.md:775]: Accidentally got staged into this PR instead of #1685.

**Review by LPardue — APPROVED** [review]
LGTM

File: draft-ietf-httpbis-digest-headers.md (hunk 1 above)
Comment (ioggstream) [reply draft-ietf-httpbis-digest-headers.md:775]: Ok, I'll rebase soon.

**Comment by ioggstream** [comment]
Rebased in #1695

**Pull request closed without merging**